### PR TITLE
Expose const opcode data in the library

### PR DESCRIPTION
**Pull request by ccuser44**
This is useful for a variety of reasons

### CODE REVIEW

**Comment by ghost** [comment]
What are the use cases here? Also there is an extra comma in the commit I think.

**Comment by ccuser44** [comment]
> What are the use cases here? Also there is an extra comma in the commit I think.

Lets say you decode the protos via `bc_to_state` and want to do something to them then it may be harder because of the remapping and stuff.

File: source.lua [hunk 1]
```diff
@@ -1076,7 +1076,7 @@ end
 return {
 	bc_to_state = lua_bc_to_state,
 	wrap_state = lua_wrap_state,,
```
Comment (ghost) [inline source.lua]: Should probably remove the extra comma here and maybe rebase.

Comment (ccuser44) [reply source.lua]: Done

**Comment by ghost** [comment]
Looks good.

**Pull request merged**